test/extended/router/gatewayapicontroller.go

```diff
@@ -370,6 +360,62 @@ func buildGateway(name, namespace, gcname, fromNs, domain string) *gatewayapiv1.
 	}
 }
 
+func waitForLoadbalancerReady(oc *exutil.CLI, gw, gwServiceName string) {
+	// check gateway LB service, note that External-IP might be hostname (AWS) or IP (Azure/GCP)
+	var lbAddress string
+	gwapiClient := gatewayapiclientset.NewForConfigOrDie(oc.AdminConfig())
+	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 10*time.Minute, false, func(context context.Context) (bool, error) {
```

Comment:
I think it makes sense to have a constant for this timeout value too (similar to the dns resolution). Because this timeout is kinda infrastructure dependent.

Reply:
done